pkg/psmdb/pmm.go

```diff
@@ -332,8 +332,7 @@ func AddPMMContainer(cr *api.PerconaServerMongoDB, secret *corev1.Secret, custom
 	}
 
 	if !cr.Spec.PMM.HasSecret(secret) {
-		log.Info(fmt.Sprintf(`Can't enable PMM: "%s" or "%s" with "%s" keys don't exist in the %s, or %s and %s secrets are out of sync`,
-			api.PMMAPIKey, api.PMMUserKey, api.PMMPasswordKey, cr.Spec.Secrets.Users, cr.Spec.Secrets.Users, api.InternalUserSecretName(cr)))
+		log.Info("Can't enable PMM, credentials are not setup correctly or secrets and internal secrets are our of sync")
```

Comment:
As for me, the previous message was more informative. Now we say that something is wrong but the user can't understand what exactly.   

@inelpandzic In general if you want to change the log message you need to confirm it with @spron-in and with @fiowro.

Reply:
Even though I'm not 100% certain and I wanted to see what you will say, but for me, I think this is enough, for the details about the correct setup user can reference our docs.

And why can't we change our logs, I think we should do it on our own, we are not breaking anything by changing them.

Reply:
@hors I agree with Inel, why do we need approval from Sergey or Dmitriy for changing log messages? We can get their input but it should be our decision.

Having said that, I agree with @hors on this message. I didn't review the changes carefully enough. Previous one was informative. Does the user know what should they do with the new message? Docs help?

Reply:
Like @hors said, current docs would probably not help. And like I said, I wasn't 100% sure about it and wanted your opinion and now I can agree that we need to enrich it.

Reply:
[fdbcefb](https://github.com/percona/percona-server-mongodb-operator/pull/1118/commits/fdbcefbe35596827a3c780c7c020b17d520ff6e7)